calculator/components/Button.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FunctionComponent } from "react";
+import { Digit, Operator } from "../types";
+export type ButtonValue = Operator | Digit | "clear";
+
+interface ButtonProps {
+  color: string;
+  value: ButtonValue;
+  onClick: () => void;
+}
+
+const Button: FunctionComponent<ButtonProps> = ({
+  color,
+  value,
+  onClick,
+}: ButtonProps) => {
+  return (
+    <button
+      className={`flex-auto w-1/4 h-14 border-none bg-${color}-600 hover:bg-${color}-400 text-2xl text-white font-semibold uppercase cursor-pointer rounded-lg outline-none`}
```

Comment:
I got it! I'll move them up to the parent. Thank you!
Also, for this one, I have an issue: when passing `color` as `"lime"` value, the button color works normally, but when passing another color argument like `violet` or `red`..., the button color disappears.

Reply:
there are 2 possible issues
1. not all colors are included in tailwind by default (see https://v2.tailwindcss.com/docs/customizing-colors#curating-colors)
2. you are trying to override color by having multiple color classes like `bg-blue-600 bg-red-600` (this should show blue)
  - i think css rules takes the first one if it see multiples, but it is generally not a good idea of having multiple color classes because the behaviour is misleading